pkg/volume/csi/nodeinfomanager/nodeinfomanager_test.go

```diff
@@ -627,6 +658,17 @@ func test(t *testing.T, addNodeInfo bool, csiNodeInfoEnabled bool, testcases []t
 			continue
 		}
 
+		// We are testing max volume limits
+		if tc.inputVolumeLimit > 0 {
```

Comment:
nit: Is it possible to have the logic below run for every test case, and have `getVolumeLimits()` return 0 if the attachKey doesn't exist? IMO, intuitively setting `inputVolumeLimit` in a test case means this value is passed into the method being tested as an input, whereas `RemoveInfo` doesn't depend on that param.